packages/ui-extensions/src/surfaces/customer-account/components/Page/examples/basic-Page-js.example.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  Page,
+  Button,
+  extension
+} from '@shopify/ui-extensions/customer-account';
+
+export default extension(
+  'customer-account.page.render',
+  (root, api) => {
+    renderApp(root, api);
+  },
+)
+
+async function renderApp(root, api) {
+  const primaryAction = root.createFragment();
+  await primaryAction.append(
+    root.createComponent(
+      Button,
+      {
+        onPress: () => {
+          console.log('Secondary action 1');
+        },
+      },
+      'Secondary action 1',
+    ),
+  );
+  const secondaryActions = root.createFragment();
+  await secondaryActions.append(
+    root.createComponent(
+      Button,
+      {
+        onPress: () => {
+          console.log('secondary action 2');
+        },
+      },
+      'Secondary action 2',
+    ),
+  );
+  await secondaryActions.append(
+    root.createComponent(
+      Button,
+      {
+        onPress: () => {
+          console.log('secondary action 3');
+        },
+      },
+      'Secondary action 3',
+    ),
+  );
+
+  const breadcrumbActions = root.createFragment();
+  await breadcrumbActions.append(
+    root.createComponent(Button, {
+      accessibilityLabel: 'Button',
+      onPress: () => {},
```

Comment:
These are still `onPress`? I thought these have all been updated to `onClick`?

Reply:
Button has been updated @brianshen1990 that being said, we dont have checkout's changes here yet and since this is a checkout component. We should probably update this soon especially once their changes are here